src/test/java/api/requests/scenarios/LoanDueDatesAfterRecallTests.java

```diff
@@ -439,7 +441,12 @@ public void initialLoanDueDateOnCreateWithPrexistingRequests()
 
     final JsonObject storedLoan = loansStorageClient.getById(loan.getId()).getJson();
 
-    final String expectedDueDate = ClockManager.getClockManager().getDateTime().plusMonths(2).toString(ISODateTimeFormat.dateTime());
+    final String expectedDueDate = ClockManager
+      .getClockManager()
+      .getDateTime()
+      .plusMonths(2)
+      .withTime(LocalTime.MIDNIGHT.minusSeconds(1))
+      .toString(ISODateTimeFormat.dateTime());
     assertThat("due date is not the recall due date (2 months)",
```

Comment:
I think this is more readable if there is whitespace between the variable and the assertion

Reply:
Whitespace added